packages/@okta/vuepress-site/docs/guides/client-secret-rotation-key/main/index.md

```diff
@@ -20,11 +20,11 @@ This guide shows you how to rotate and manage your client secrets without servic
 #### What you need
 
 * [Okta Developer Edition organization](https://developer.okta.com/signup)
-* **OAuth secrets and key management** feature enabled in the Admin Console (**Settings** | **Features**)
-* Access to the new APIs: Client Secret Management `/api/v1/apps/{appId}//credentials/secrets` and JWKS Management `/api/v1/apps/{appId}/credentials/jwks`
+* **OAuth secrets and key management** enabled in the Admin Console (**Settings** | **Features**)
```

Comment:
this is no longer self-service EA - so this line actually needs to be removed. client secret rotation is GA and has been since 2022. EEKS